packages/cli/source/tsconfig.dev.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
do we need to have one tsconfig.dev and tsconfig.build in the core/ folder?

Reply:
https://github.com/coveo/cli/pull/913/files#r964405952

Yep, this tsconfig is used by oclif in dev mode, aka `bin/dev`.